cmd/example-tests/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"sort"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	root := &cobra.Command{
+		Long: "OpenShift Tests External Binary Example",
```

Comment:
```suggestion
		Long: "OpenShift Tests Extension Example",
```
I think using "Extension" consistently will help people understand the concept better and age a bit better as mechanisms changes.